code/game/gamemodes/cult/cult.dm

```diff
@@ -38,8 +38,8 @@
 	restricted_jobs = list("Chaplain","AI", "Cyborg", "Security Officer", "Warden", "Detective", "Head of Security", "Captain", "Head of Personnel")
 	protected_jobs = list()
 	required_players = 24
-	required_enemies = 4
-	recommended_enemies = 4
+	required_enemies = 3
+	recommended_enemies = 3
```

Comment:
undocumented?

Reply:
this was IIRC a fix for a situation where the math worked in a funny way to prevent the gamemode from launching

Reply:
I can remove it or I can add a changelog, won't put it in the CL though unless I understand what exactly it's there for.